shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -165,6 +166,10 @@ - (BOOL)isAccessibilityElement {
   //  We enforce in the framework that no other useful semantics are merged with these nodes.
   if ([self node].HasFlag(blink::SemanticsFlags::kScopesRoute))
     return false;
+  // If the node is the placeholder for a platform view. We know it should be a container.
```

Comment:
Are you adding the asserts for this in the framework?

I would probably remove the comment and just add this case to this if above. It's just anther node that doesn't get semantics merged in.